src/graphics/aurora/highlightable.h

```diff
@@ -76,6 +78,8 @@ class Highlightable {
 	float _lowerBoundG;
 	float _lowerBoundB;
 	float _lowerBoundA;
+	
```

Comment:
Also, to catch these, you could configure your editor/IDE to show trailing whitespace (i.e. tabs and spaces at the end of lines) and/or show all white space.

This is how that section looks for me:
![foo](https://user-images.githubusercontent.com/156130/29494383-a67a27cc-85a9-11e7-9e19-3115d162e779.png)